### ISSUE TITLE
Syntax highlighting is broken on 1.4.0 

### ISSUE THREAD

**Issue by wsmd** [atom/base16-tomorrow-dark-theme#33]
I just updated to 1.4.0 few hours ago and it broke syntax highlighting for all languages:

![screen shot 2016-10-17 at 11 11 18 am](https://cloud.githubusercontent.com/assets/2100222/19443598/34185946-945b-11e6-88e4-b947c8d934ac.png)

![screen shot 2016-10-17 at 11 13 41 am](https://cloud.githubusercontent.com/assets/2100222/19443584/2c290762-945b-11e6-8899-f104d76ea418.png)

![screen shot 2016-10-17 at 11 16 03 am](https://cloud.githubusercontent.com/assets/2100222/19443565/2030f578-945b-11e6-9232-d5059f8bfeae.png)

No styles are applied in `~/.atom/styles.less`.

**Comment by Ben3eeE** [comment]
I believe this is expected because of the shadow DOM changes. Don't manually update bundled packages unless you know what you're doing.

**Comment by wsmd** [comment]
> I believe this is expected because of the shadow DOM changes.

Can elaborate more?

> Don't manually update bundled packages unless you know what you're doing.

I didn't update it manually. It showed up under updates this morning and I clicked "update".

**Comment by Ben3eeE** [comment]
> Can you elaborate more?

Sorry for the quick answer and close. We and in progress of removing the shadow DOM, so bascially all syntax/theme packages are having a breaking update at the moment, see https://github.com/atom/atom/pull/12903 for a perfect explanation by @as-cii about this change. These updates will not work until this PR is merged.

> I didn't update it manually. It showed up under updates this morning and I clicked "update".

This is because you some time in the past manually updated it causing Atom to give you update notifications, or you are using a package for updating which also can cause this. We just closed https://github.com/atom/one-dark-syntax/issues/80#issuecomment-254317635 that was caused by a package updating bundled packages as well. Bundled packages can have breaking changes like this and are only supposed to be upgraded together with Atom because of these changes.
